toniepodcastsync.py

```diff
@@ -197,3 +206,30 @@ def __generate_chapter_title(self, ep: Episode) -> str:
     def __is_tonie_empty(self, tonie_id: str) -> bool:
         tonie = self.__tonieDict[tonie_id]
         return tonie.chaptersPresent == 0
+
+    def __adjust_volume__(self, audio_bytes: bytes, volume_adjustment: int) -> bytes:
+        audio = AudioSegment.from_file(BytesIO(audio_bytes), format="mp3")
+
+        adjusted_audio = audio + volume_adjustment
+
+        byte_io = BytesIO()
+
+        adjusted_audio.export(byte_io, format="mp3")
+
+        return byte_io.getvalue()
+
+    def __is_ffmpeg_available(self) -> bool:
+        try:
+            # Safe to use untrusted input: executable is hardcoded
+            # noqa: S603
+            executable = "ffmpeg" if platform.system().lower() != "windows" else "ffmpeg.exe"
```

Comment:
Ah, nice that you considered platform-independent'ness! I was wondering if this works on Windows as well, as I noticed another user running this on a Windows host. Unfortunately, I have no option to test it on Windows...
Do you have a Windows setup where you could try it out already?

Reply:
No, I can't try it either. To be honest, I only asked Chat GPT to make it Windows safe.